.eslintrc

```diff
@@ -13,6 +13,7 @@
   },
   "extends": "airbnb",
   "rules": {
+    "no-return-assign": 0,
```

Comment:
Ah, thanks @borisyankov. Doing it that way made the linter unhappy for other reasons (`block-spacing` rules, etc.) but I think I've got it now, without making any changes to `.eslintrc`.